tests-utils/src/main/scala/tech/beshu/ror/utils/containers/images/Elasticsearch.scala

```diff
@@ -174,23 +257,20 @@ class Elasticsearch(esVersion: String,
       .foldLeft(builder) { case (currentBuilder, update) => update(currentBuilder) }
   }
 
-  private def esConfigFileBasedOn(config: Config,
-                                  withEsConfigBuilder: EsConfigBuilder => EsConfigBuilder) = {
+  def esConfigFile(networkHost: String): File = {
```

Comment:
I'm not sure about this public method. Above, we have `val config: Config` which describes how the ES config looks (we create the elasticsearch.yml from this description). Can we not add the `networkHost` there?